### PR TITLE
Fixing the Modify Monitored Item response

### PR DESCRIPTION
**Pull request by Johen8**
I'm not sure if I'm doing this right, I'm kind of new to GitHub. But, as mentioned in #175 , the Modify Monitored Item response was wrong and it was actually a simple fix, as you can see. It took me some hours of debugging until I found that wrong line.

### CODE REVIEW

**Comment by Johen8** [comment]
Sorry for the 'revert' of the 'revert', as I said, I got a little tangled up with GitHub.

**Comment by oroulet** [comment]
Stupide bug! Thanks! Take a backup then try 
Git rebase -I xxx to merge the last 3 commits
Git commit -- amend # to modify commit text
Git push -force
That should fix it

File: opcua/server/internal_subscription.py [hunk 1]
```diff
@@ -59,15 +59,16 @@ def trigger_datachange(self, handle, nodeid, attr):
     def _modify_monitored_item(self, params):
         with self._lock:
             for mdata in self._monitored_items.values():
-                result = ua.MonitoredItemCreateResult()
+                result = ua.MonitoredItemModifyResult() # main thing that was wrong
```
Comment (destogl) [inline opcua/server/internal_subscription.py]: Before merging comment should be removed.

Also I am not sure that comments under are also really needed...

**Comment by Johen8** [comment]
I'm really sorry, but I'm using SourceTree and I'm not that used to command line GitHub. What are the parameters I have to pass to rebase and commit? I should run these commands on the master branch of my fork, right?

**Comment by zerox1212** [comment]
If you aren't actively working in your git fork it might be easier to just delete your git hub repository and then fork it again. Push these minor changes then make a PR... at least that is what I do because Git Desktop sucks and I don't know command line either.

**Comment by oroulet** [comment]
remove the comments. i can easily squash your commits in one. But your should try rebase, it is interesting to know.

**Comment by Johen8** [comment]
I'm sorry @oroulet , I was in the middle of creating the new repo like zero said when I saw your comment.

**Pull request closed without merging**